### PR TITLE
Alter disclaimer text for unauthenticated access.

### PR DESCRIPTION
**Pull request by dcadams**

### CODE REVIEW

File: templates/courseware/theme-course_info_extra_sneakpeek.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<%! from django.utils.translation import ugettext as _ %>
+
+<div class="unauth-tos">
+  ${_("By exploring the course, you are agreeing to our {tos_link_start}Terms of Service{link_end} and {privacy_link_start}Privacy Policy{link_end}. Please read them carefully.")\
+    .format(tos_link_start='<a href="https://lagunita.stanford.edu/tos">',
```
Comment (stvstnfrd) [inline templates/courseware/theme-course_info_extra_sneakpeek.html]: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines

**Comment by stvstnfrd** [comment]
@caesar2164 Do we need a copy of this in the other themes? Will the include break w/out it?

**Comment by caesar2164** [comment]
hmm, we could just use the optional inlcude

**Comment by caesar2164** [comment]
I'd say we rename this file `course_info_extra_sneakpeek.html` and we add it with an optional include...

**Comment by stvstnfrd** [comment]
I believe we said F2F that this is no longer needed.

**Pull request closed without merging**